chore(docs): Migrate the callouts to bullets (#3580)
kroxylicious/kroxylicious: kroxylicious-docs/docs/_modules/oauth-bearer-validation/con-example-kafkaprotocolfilter-resource.adoc

spec:
  type: OauthBearerValidation
  configTemplate:
      jwksEndpointUrl: https://oauth/JWKS
      jwksEndpointRefreshMs: 3600000
      jwksEndpointRetryBackoffMs: 100
      jwksEndpointRetryBackoffMaxMs: 10000
      scopeClaimName: scope
      subClaimName: sub
      authenticateBackOffMaxMs: 60000
      authenticateCacheMaxSize: 1000
      expectedAudience: https://first.audience, https//second.audience
      expectedIssuer: https://your-domain.auth/
----
where:

* `jwksEndpointUrl` is the OAuth/OIDC provider URL from which the provider's JWKS (JSON Web Key Set) can be retrieved.
* `jwksEndpointRefreshMs` is the (optional) value in milliseconds for the broker to wait between refreshing its JWKS (JSON Web Key Set) cache that contains the keys to verify the signature of the JWT.
* `jwksEndpointRetryBackoffMs` is the (optional) value in milliseconds for the initial wait between JWKS (JSON Web Key Set) retrieval attempts from the external authentication provider.

> * `jwksEndpointRetryBackoffMaxMs` is the (optional) value in milliseconds for the maximum wait between attempts to retrieve the JWKS (JSON Web Key Set) from the external authentication provider.

Reviewer: ```suggestion
* `jwksEndpointRetryBackoffMaxMs` specifies the maximum delay, in milliseconds, between JWKS retrieval attempts.
```
